Index: app/src/test/java/edu/byui/childrenlearningapp/LettersTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/test/java/edu/byui/childrenlearningapp/LettersTest.java	(date 1614212182148)
+++ app/src/test/java/edu/byui/childrenlearningapp/LettersTest.java	(date 1614212182148)
@@ -0,0 +1,11 @@
+package edu.byui.childrenlearningapp;
+
+import org.junit.Test;
+
+public class LettersTest {
+    @Test
+    public void whatervername(){
+
+    }
+
+}
