Index: app/src/test/java/edu/byui/childrenlearningapp/LettersTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package edu.byui.childrenlearningapp;\r\n\r\nimport org.junit.Test;\r\n\r\nimport static org.junit.Assert.*;\r\n\r\npublic class LettersTest {\r\n\r\n    @Test\r\n    public void getName() {\r\n    }\r\n\r\n    @Test\r\n    public void getVowel() {\r\n        Letters letter = new Letters();\r\n        assertEquals(letter.getVowel(),false);\r\n        letter.setVowel(true);\r\n        assertEquals(letter.getVowel(),true);\r\n    }\r\n\r\n    @Test\r\n    public void setName() {\r\n    }\r\n\r\n    @Test\r\n    public void setVowel() {\r\n    }\r\n\r\n    @Test\r\n    public void playSound() {\r\n    }\r\n\r\n    @Test\r\n    public void onCreateDisplay() {\r\n    }\r\n\r\n    @Test\r\n    public void onPress() {\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/test/java/edu/byui/childrenlearningapp/LettersTest.java	(revision 1a6f6c154b86dee3134f518aca8ede58abd47a02)
+++ app/src/test/java/edu/byui/childrenlearningapp/LettersTest.java	(date 1614214099006)
@@ -8,6 +8,10 @@
 
     @Test
     public void getName() {
+        Letters letter = new Letters();
+        assertEquals(null, letter.getName());
+        letter.setName("Frani");
+        assertEquals("Frani",letter.getName());
     }
 
     @Test
Index: app/src/main/java/edu/byui/childrenlearningapp/Child.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/edu/byui/childrenlearningapp/Child.java	(date 1614814080328)
+++ app/src/main/java/edu/byui/childrenlearningapp/Child.java	(date 1614814080328)
@@ -0,0 +1,23 @@
+package edu.byui.childrenlearningapp;
+
+public class Child {
+
+    private String name;
+    private int age;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+}
